README.md

```diff
@@ -1,153 +1,146 @@
+A typical response may look like this:
+```bash
+|NAME               |AGE
+|redis-enterprise   |5m
+```
 
-5) The next step applies crd.yaml, creating a CustomResourceDefinition for redis enterprise cluster resource.
-This creates another API resource to be handled by the k8s API server and managed by the operator we will deploy next.
-    ```
-    kubectl apply -f crd.yaml
-    ```
+### OpenShift
 
-    > You should receive the following response:
+The "OpenShift" installations deploys the operator from the current release with the RHEL image from DockerHub and default OpenShift settings.
+This is the fastest way to get up and running with a new cluster on OpenShift 3.x.
+For OpenShift 4.x, you may choose to use OLM deployment from within your OpenShift cluster or follow the steps below.
+Other custom configurations are referenced in this repository.
+> Note: you will need to replace `<my-project>` with your project name
 
-    `customresourcedefinition.apiextensions.k8s.io/redisenterpriseclusters.app.redislabs.com configured`
+1. Create a new project:
 
-6) Create the operator deployment: a deployment responsible for managing the k8s deployment and lifecycle of a redis-enterprise-cluster.
-    Among many other responsibilities, it creates a stateful set that runs the redis enterprise nodes (as pods).
+```bash
+oc new-project my-project
+```
 
-    Before applying, edit the tag according to the relevant operator version: `image: redislabs/operator:tag`
-    ```
-    kubectl apply -f operator.yaml
-    ```
+2. Perform the following commands (you need cluster admin permissions for your Kubernetes cluster):
 
-    > You should receive the following response:
+```bash
+oc apply -f openshift/scc.yaml
+```
 
-    `deployment.apps/redis-enterprise-operator created`
+You should receive the following response:
+`securitycontextconstraints.security.openshift.io "redis-enterprise-scc" configured`
 
-7) Run `kubectl get Deployment` and verify redis-enterprise-operator deployment is running.
+3. Provide the operator permissions for pods (substitute your project for "my-project"):
 
-    A typical response may look like this:
-    ```
-    |NAME                     |DESIRED | CURRENT  | UP-TO-DATE | AVAILABLE | AGE|
-    |-------------------------|-------------------------------------------------|
-    |redis-enterprise-operator|1	   | 1        |  1         | 1         | 2m |
-    ```
+```bash
+oc adm policy add-scc-to-group redis-enterprise-scc system:serviceaccounts:my-project
+```
 
-8)  Create A Redis Enterprise Cluster.  Choose the configuration relevant for you (see next section).  There are additional examples in the examples folder. Note that you need to specify an image tag if you'd like to pull a RHEL image.
+4. Deploy the OpenShift operator bundle:
 
-    ```
-    kubectl apply -f redis-enterprise-cluster.yaml
-    ```
+```bash
+kubectl apply -f openshift.bundle.yaml
```

Comment:
oc apply

Reply:
we can, but don't have to since we oc applied the scc.yaml separately.